tools/mqtt-bench/bench.go

```diff
@@ -44,14 +44,14 @@ func Benchmark(cfg Config) error {
 		caByte, _ = io.ReadAll(caFile)
 	}
 
-	data, err := os.ReadFile(cfg.Mg.ConnFile)
+	data, err := os.ReadFile(cfg.Smq.ConnFile)
 	if err != nil {
 		return fmt.Errorf("error loading connections file: %s", err)
 	}
 
-	mg := magistrala{}
+	mg := superMQ{}
```

Comment:
please change `mg` to `smq`

In this same file we have word `mgChan` and `mgCli`